house_screen.py

```diff
@@ -4,7 +4,6 @@
 from game_util import PetConfig as config
 from pet_selection import PetSelection
 from game_over import GameOver
-from pygame.locals import *
```

Comment:
I removed this import because it would not be necessary for us to import the whole module, would it?🤔